### ISSUE TITLE
In Admin User Edit "May Import Local Repository" always editable

### ISSUE THREAD

**Issue by EnricoFerro** [go-gitea/gitea#4779]
- Gitea version (or commit ref): Actual ( Probably from d67b278 )
- Git version:
- Operating system:
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No
  - [ ] Not relevant
- Log gist:

## Description
**May Import Local Repositry** remain editable when setting IMPORT_LOCAL_PATHS is false or missing, 
should be disabled like **May Create Git Hooks**  #2450 ( with the setting DISABLE_GIT_HOOKS ) 

## Screenshots

Setting in the screenshot:  
```
[security]
...
IMPORT_LOCAL_PATHS = false
DISABLE_GIT_HOOKS = true 
```

As is:
![image](https://user-images.githubusercontent.com/25525316/44553918-6b484500-a72f-11e8-8602-86a4b991d7ad.png)

How should be:
![image](https://user-images.githubusercontent.com/25525316/44554045-c4b07400-a72f-11e8-8a76-86c4af0e64fb.png)

**Comment by techknowlogick** [comment]
Link ticket #4482

**Comment by EnricoFerro** [comment]
Merged #4780 that resolve this issue
